src/expr.rs

```diff
@@ -678,13 +679,13 @@ impl<'a> Rewrite for Loop<'a> {
         self.block
             .rewrite(context, width, offset)
             .map(|result| {
-                format!("{}{}{}{}{}",
-                        label_string,
-                        self.keyword,
-                        pat_expr_string,
-                        block_sep,
-                        result)
-            })
+            format!("{}{}{}{}{}",
```

Comment:
This seems bad - we're indenting the closure body the same as the surrounding chain (which would be more left-floating if `self` we're a longer name. Also the closing brace and paren seem way too far to the left.